benchmark/CMakeLists.txt

```
@@ -1,6 +1,6 @@
 add_executable(ompl_benchmark
     CFGBenchmark.cpp BenchmarkOptions.cpp BenchmarkTypes.cpp benchmark.cpp)
-target_link_libraries(ompl_benchmark ${OMPLAPP_LIBRARIES} ompl ompl_app_base ${Boost_PROGRAM_OPTIONS_LIBRARY})
+target_link_libraries(ompl_benchmark ${OMPLAPP_LIBRARIES} ompl ompl_app_base Boost::program_options)
```

```suggestion
target_link_libraries(ompl_benchmark ompl::omplapp ompl ompl_app_base Boost::program_options)
```

If you can, try linking to an ALIAS targets for all of these. They don't all exist, but are easily created for the internal ones..